modules/log/commerce_log.commerce_log_templates.yml

```diff
@@ -6,6 +6,10 @@ cart_item_removed:
   category: commerce_cart
   label: 'Removed from cart'
   template: '<p><em>{{ purchased_entity_label }}</em> removed from the cart.</p>'
+cart_item_quantity_changed:
+  category: commerce_cart
+  label: 'Quantity changed from cart'
+  template: '<p>Quantity for <em>{{ purchased_entity_label }}</em> changed in the cart form <em>{{ original_quantity }} to {{ quantity }}</em>.</p>'
```

Comment:
Change "Quantity for" to "Quantity of".
No emphasis around the quantities.
And "form" should be "from".